unified_planning/grpc/unified_planning.proto

```diff
@@ -367,39 +367,43 @@ message Problem {
 enum Feature {
     // PROBLEM_CLASS
     ACTION_BASED = 0;
+    HIERARCHICAL = 1;
     // TIME
-    CONTINUOUS_TIME = 1;
-    DISCRETE_TIME = 2;
-    INTERMEDIATE_CONDITIONS_AND_EFFECTS = 3;
-    TIMED_EFFECT = 4;
-    TIMED_GOALS = 5;
-    DURATION_INEQUALITIES = 6;
+    CONTINUOUS_TIME = 2;
+    DISCRETE_TIME = 3;
+    INTERMEDIATE_CONDITIONS_AND_EFFECTS = 4;
+    TIMED_EFFECT = 5;
+    TIMED_GOALS = 6;
+    DURATION_INEQUALITIES = 7;
+    // EXPRESSION_DURATION
+    STATIC_FLUENTS = 8;
+    FLUENTS = 9;
```

Comment:
these names can be confusing, can we call them `DUARTION_FLUENTS` or something like that?